### ISSUE TITLE
xess.pro

### ISSUE THREAD

**Issue by xXtenmaXx** [reek/anti-adblock-killer#3359]
please add this site, is detecting my adblock

**Comment by uBlock-user** [comment]
Test link - http://xess.pro/q4Y5w9

http://xess.pro/js/ads.js

@jspenguin2017

**Comment by jspenguin2017** [comment]
Malvertizing popup, must kill...

**Comment by jspenguin2017** [comment]
How can JavaScript control my cursor when it is outside of the page frame... 

Is this the trick they used? https://blog.malwarebytes.com/cybercrime/2016/11/forced-into-installing-a-chrome-extension/

There needs to be a force close shortcut, somehow `Shift+Esc` (Chrome task manager) can be blocked by the website...

**Comment by xXtenmaXx** [comment]
@jspenguin2017 @uBlock-user is weird, doesn't work with this one https://xess.pro/aLfWx5Ao
i have ublock origin + anti-adblock killer + adsbypasser
i tried disabling adsbypasser but is the same

**Comment by jspenguin2017** [comment]
I can't patch AAK.

**Comment by uBlock-user** [comment]
@xXtenmaXx Remove AAK and install uBlockProtector.

**Comment by jspenguin2017** [comment]
Is this malvertizing campaign using some sort of exploit? It is so hard to get out. I had to use several tricks to get into the task manager to crash the tab. 
I managed to dump the page source here: https://gist.github.com/jspenguin2017/9702b3401e21424c0b660d12cc7947ef

**Comment by uBlock-user** [comment]
Is this the popup script - https://xess.pro/js/popup.js ?

**Comment by xXtenmaXx** [comment]
@jspenguin2017 @uBlock-user thanks for your help. My English is terrible and I don't know anything about how scripts works. I will try ublock protector

**Comment by jspenguin2017** [comment]
@uBlock-user Yes, that is the popup script, it's one of those popup script, but the malvertizing page it redirect to can somehow make my mouse act weird.

**Comment by uBlock-user** [comment]
I get a popup and it's then blocked by uBO, Malvertising shit never loads for me.

**Comment by jspenguin2017** [comment]
I want to know how they did it, that's all.
